packages/fxa-auth-server/lib/routes/oauth/client/get.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const hex = require('buf').to.hex;
+const Joi = require('@hapi/joi');
+
+const AppError = require('../../../oauth/error');
+const db = require('../../../oauth/db');
+const validators = require('../../../oauth/validators');
+
+module.exports = (log) => ({
```

Comment:
I wonder if we should move the `db` require into `index.js` and pass it in like `log`. I think eventually `oauth/db` will change to initialize after require, i.e. not export `new OauthDB()`

Reply:
I moved the require into `index.js` and did some renaming.  Please take another quick look.  Thanks!